### ISSUE TITLE
Can I work on this?

### ISSUE THREAD

**Issue by soywiz** [DragonBones/DragonBonesJAVA#3]
Sometime ago I ported dragonbones to Java: https://github.com/soywiz/DragonBonesJava
I have seen that there are other efforts: https://github.com/GDoom/DragonBonesJava https://github.com/DragonBones/DragonBonesJAVA/issues/1
But they are all outdated.

I can try to port the current version of DragonBones from C# to Java.
https://github.com/DragonBones/DragonBonesCSharp

In order to be able to update it, what I'm going to do is to copy C# folder into my repo. And then I will copy C# files in Java files and will start changing syntax file by file.
Since I have the same C# and Java code in the same repo, if C# is updated, I can just copy all files here and check the diff to apply it to Java version, which should make updating easier. Also I would add a .travis.yml to compile java version to show how to compile and test it.

If nobody is working on this, I will work on it, and will make a PR soon probably in a few days.

Regards, and thank you for your awesome work!

**Comment by akdcl** [comment]
Thank you for your contribution :)
If you are willing to try, it is recommended to proceed from the TypeScript version, because the C# is not up to date.
https://github.com/DragonBones/DragonBonesJS

**Comment by soywiz** [comment]
Understood! And thanks for replying :) I will start later today. Then probably I will code it in kotlin that is JVM based too, and syntax is much more similar to typescript than java and will end much faster. I will use last master from github. I hope it is stable!

**Comment by tpnet** [comment]
@soywiz  Hello，  How can i run this ,sample is empty

**Comment by soywiz** [comment]
I only implemented the backend part (all the arithmetic and logic); not implemented rendering.
A possible frontend for rendering would be libgdx.

**Comment by Sashie** [comment]
did you ever finish work on this? I still don't see any examples for libgdx

**Comment by soywiz** [comment]
Nope. But this week I have ported (TypeScript master) to Kotlin (that is much more similar to typescript) and my game engine instead: https://soywiz.com/ported-dragonbones-to-korge/

The code lives here: https://github.com/korlibs/korlibs/tree/master/korge/korge-dragonbones/common
If you don't mind to include the kotlin runtime, you can adapt the code to libgdx too.

**Comment by Sashie** [comment]
oh fancy, that saves me the trouble of attempting to port the latest version over <3 thanks
